### ISSUE TITLE
size of tensor mismatch in webqsp_full

### ISSUE THREAD

**Issue by cyrdocoo** [malllabiisc/EmbedKGQA#48]
![image](https://user-images.githubusercontent.com/52520046/98094433-4087d880-1ec4-11eb-962a-aff3faa2e2e9.png)
The command i used is :
'python main.py --mode train --relation_dim 200 --do_batch_norm 0 \
--gpu 0 --freeze 1 --batch_size 16 --validate_every 10 --hops webqsp_full --lr 0.00002 --entdrop 0.0 --reldrop 0.0 --scoredrop 0.0 \
--decay 1.0 --model ComplEx --patience 20 --ls 0.0 --l3_reg 0.001 --nb_epochs 200 --outfile full_fbwq'

**Comment by apoorvumang** [comment]
Can you try to use parameter --relation_dim 128 and let me know the results?

**Comment by cyrdocoo** [comment]
> Can you try to use parameter --relation_dim 128 and let me know the results?

Thanks, I tried the parameter and got:
'webqsp_full hop Validation accuracy (no relation scoring) increased from previous epoch 0.5425048669695003'
By the way, I was wondering why the model with 2-hop filtering performs so good but you don't take it as the baseline?

**Comment by apoorvumang** [comment]
> By the way, I am wondering why the model with 2-hop filtering performed so good but you didn't take it as the baseline?

We wanted to focus on the incomplete KG setting where our model outperforms the others (in MetaQA). In such cases, neighbourhood filtering is not a good option since we may miss out on answer nodes due to graph incompleteness. So we reported our result as the one without neighbourhood filtering. As you can see in the ablation study, on the incomplete graph, the model with 2-hop filtering performs worse than the one without filtering, reaffirming our point.
